### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by gipert**

### CODE REVIEW

**Review by ManuelHu — APPROVED** [review]

**Comment by gipert** [comment]
I started with the tutorial (but want to add much more in the next days), please have a look.

@ManuelHu  @tdixon97

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+![Geometry visualization](img/tutorial-pyg4-view.jpg)
+
+By following instructions in the <project:./install.rst> section, you should
+have access to the `remage` executable. We are now ready to simulate some
+particle physics with it.
+
+Like any other Geant4-based application, we need to configure the simulation
+with a macro file. Standard Geant4 commands as well as custom commands (see the
+<project:./rmg-commands.rst>) are available.
```
Comment (ManuelHu) [inline docs/tutorial.md:118]: those `project:./` links do not work (at least on RTD, did not try locally)

File: docs/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+/gps/energy 1000 keV
+
+/run/beamOn 50
+```
+
+We can finally pass the GDML geometry and the macro to the `remage` executable
+and look at the result!
+
+```console
+$ remage --interactive --gdml-files geometry.gdml vis-gammas.mac
```
Comment (ManuelHu) [inline docs/tutorial.md:179]: maybe add the option to select an output file? (also below in the note block). Or do you plan to have this in the next (not yet written) part?

File: docs/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+Like any other Geant4-based application, we need to configure the simulation
+with a macro file. Standard Geant4 commands as well as custom commands (see the
+<project:./rmg-commands.rst>) are available.
+
+At the beginning of the file, we have to register the "sensitive detectors" (in
+our simple case, the two HPGes and the LAr). *remage* offers several types of
+predefined detectors, targeting different physical quantities of the particles
+that interact with them. HPGes are of type `Germanium`, while the LAr is of
+type `scintillator`. Their difference in terms of simulation output will be
```
Comment (ManuelHu) [inline docs/tutorial.md:124]:
```suggestion
type `Scintillator`. Their difference in terms of simulation output will be
```

File: docs/tutorial.md (hunk 1 above)
Comment (gipert) [reply docs/tutorial.md:118]: uff works locally...

**Comment by gipert** [comment]
I think this looks better now. Anything else crucial I should include?

**Comment by ManuelHu** [comment]
Maybe we should use a less confusing (i.e. more useful in the field of particle detectors) material for the source volume?

Apart from that it already looks very good :-)

**Comment by tdixon97** [comment]
I like the source material

**Comment by gipert** [comment]
Trying to fix #149 but it's a bit flaky, it does not recognize parameter description fields yet...

**Comment by ManuelHu** [comment]
maybe we should move this to a seperate PR?

**Pull request merged**